lib/config/options/index.ts

```diff
@@ -1623,6 +1623,14 @@ const options: RenovateOptions[] = [
     default: true,
     supportedPlatforms: ['bitbucket', 'bitbucket-server'],
   },
+  {
+    name: 'bbUseDevelopmentBranch',
```

Comment:
I think this needs to be `globalOnly` because by the time config is read from the repo, it would be too late.

Reply:
The mainBranch / developmentBranch logic is currently within the `initRepo` phase.  

Wouldn't this mean that `bbUseDevelopmentBranch` can be used as either a global config or a repository config?

Reply:
initRepo is done prior to reading config